README.rst

```diff
@@ -28,7 +28,11 @@ By default, on apt 1.0 and below, the booted kernel, the latest-installed kernel
 Ubuntu has multiple suggestions on how to remove kernels:
   https://help.ubuntu.com/community/RemoveOldKernels
 
-A good recommendation is using unattendend-upgrades(`debian <https://wiki.debian.org/UnattendedUpgrades>`__, `ubuntu <https://help.ubuntu.com/community/AutomaticSecurityUpdates>`__) as it has an option to remove unused dependencies, but that applies to all packages not just kernels.
+A great recommendation is to make use of unattended-upgrades `u-u` (`debian <https://wiki.debian.org/UnattendedUpgrades>`__, `ubuntu <https://help.ubuntu.com/community/AutomaticSecurityUpdates>`__). Since version **1.0** (`debian <https://packages.debian.org/search?keywords=unattended-upgrades>`__, `ubuntu <https://packages.ubuntu.com/search?keywords=unattended-upgrades>`__) *u-u* is capable of removing unused kernel packages (*Remove-Unused-Kernel-Packages*).
```

Comment:
IMO ... is capable of removing unused kernel packages ... -> ... removes unused kernel packages by default ... would be more accurate and easier to understand.

Reply:
Fixed in 37f1c64